src/Calendar.Plugin.Sample/SampleApp/ViewModels/CalendarPickerPopupViewModel.cs

```diff
@@ -35,8 +44,8 @@ public DateTime MonthYear
             set => SetProperty(ref _monthYear, value);
         }
 
-        private DateTime _selectedDate = DateTime.Today;
-        public DateTime SelectedDate
+        private DateTime? _selectedDate = DateTime.Today;
```

Comment:
im uncertain whether this is good, since this is now nullable, maybe just remove the Today value? @antonioseric